ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/enums/JurisdictionConnections.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.divorce.ccd.model.enums;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.hmcts.ccd.sdk.api.HasLabel;
+
+@Getter
+@AllArgsConstructor
+public enum JurisdictionConnections implements HasLabel {
+    @JsonProperty("A")
+    A("A"),
```

Comment:
does this have any specific meaning ? A means something B means something ? 
we can use ` @JsonPropertyDescription()` to add something to indicate what it means which may help if any one looks at the class

Reply:
Maybe the enum value could have a more explantory name for use in Java code.  Something like PET_RESP_RESIDENT("A")